lib/fog/core/model.rb

```diff
@@ -20,6 +21,10 @@ def initialize(new_attributes = {})
       merge_attributes(attribs)
     end
 
+    def cache
+      Fog::Cache.new(self)
+    end
```

Comment:
What if we expose only the `dump` and `load` methods from `Cache` instead of the whole object?
Do we need everything to be available form the model itself?